static/src/javascripts/projects/common/modules/ui/last-modified.js

```diff
@@ -0,0 +1,22 @@
+define([
+    'bean',
+    'common/utils/$'
+], function (
+    bean,
+    $
+) {
+    function expand() {
+        $('.js-lm').toggleClass('u-h');
+    }
+
+    function init() {
+        if ($('.js-lm').length !== 0) {
+            $('.js-wpd').addClass('content__dateline-wpd--modified tone-colour');
+            bean.on(document.body, 'click', '.js-wpd', expand);
```

Comment:
Why are you binding to `document.body` here, surely the parent should suffice are their more than one?